spec/features/3_users_spec.rb

```diff
@@ -60,51 +59,65 @@
   end
 
   scenario "/users/:id lists user's details", points: 1 do
-    user = FactoryGirl.create(:user)
+    user = create(:user)
     login_as(user, :scope => :user)
 
     visit "/users/#{user.id}"
 
     expect(page).to have_content(user.username)
   end
 
+  scenario "/users/:id lists user's photos caption", points: 2 do
+    user = create(:user)
+    photo_1 = FactoryBot.create(:photo, :user_id => user.id)
+    photo_2 = FactoryBot.create(:photo, :user_id => user.id)
+    login_as(user, :scope => :user)
+
+    visit "/users/#{user.id}"
+
+    photos = Photo.all
+    photos.each do |photo|
+      expect(page).to have_content(photo.caption)
+    end
+  end
+
   scenario "/users/:id lists user's photos", points: 2 do
-    user = FactoryGirl.create(:user)
-    photo_1 = FactoryGirl.create(:photo, :user_id => user.id)
-    photo_2 = FactoryGirl.create(:photo, :user_id => user.id)
+    user = create(:user)
+    photo_1 = FactoryBot.create(:photo, :user_id => user.id)
+    photo_2 = FactoryBot.create(:photo, :user_id => user.id)
     login_as(user, :scope => :user)
 
     visit "/users/#{user.id}"
 
-    expect(page).to have_content(photo_1.caption)
-    expect(page).to have_css("img[src*='#{photo_1.image}']")
-    expect(page).to have_content(photo_2.caption)
-    expect(page).to have_css("img[src*='#{photo_2.image}']")
+    photos = Photo.all
+    photos.each do |photo|
+      expect(page).to have_css("img[src*='#{photo.image}']")
+    end
   end
 
   scenario "when signed in header has link to /users/:id", points: 2 do
-    user = FactoryGirl.create(:user)
+    user = create(:user)
     login_as(user, :scope => :user)
 
     visit "/"
 
-    within("nav") {
+    within('nav') do
       expect(page).to have_link(nil, href: "/users/#{user.id}")
-    }
+    end
   end
 
   scenario "unless signed in, no link to /users/:id", points: 1 do
-    user = FactoryGirl.create(:user)
+    user = create(:user)
 
     visit "/"
 
-    within("nav") {
+    within('nav') do
       expect(page).not_to have_link(nil, href: "/users/#{user.id}")
-    }
+    end
   end
 
   scenario "in routes.rb, /users/:id below 'devise_for :users'", points: 1 do
```

Comment:
@raghubetina The spec title and actual spec is different. Could you please take a look